config/config.go

```diff
@@ -80,6 +80,9 @@ type StatusConfig struct {
 	Pass string `yaml:"pass"`
 }
 
+// DefaultTier2IPRange is the default tier2 virtual server IP range
+var DefaultTier2IPRange = "172.0.0.1/24"
```

Comment:
ParseCIDR returns (172.0.0.0, 172.0.0.0/24, nil) for this - as the /24 makes it a 172.0.0.0-255 range. We should just make it the .0 for clarity.

BTW, what are we using in systest? The 0, and 255 are special addresses, that is to say, reserved as the network ID and the broadcast ID. The BIG-IP accepts these addresses but don't know if it treats them specially.

Reply:
Systests uses 10.0.0.1/32

Reply:
Changing from .1 -> .0